.github/workflows/ci.yaml

```
@@ -29,6 +29,9 @@ jobs:
       - run: rstcheck --ignore-directives automodule $(git ls-files '*.rst')
       - run: yamllint --strict $(git ls-files '*.yaml' '*.yml')
       - run: python setup.py build_sphinx
+      - name: Check for broken links in documentation
+        run: |
+          make -C docs linkcheck
```

You didn't like my proposal? :slightly_smiling_face: 
```suggestion
        run: make -C docs linkcheck
```

It's shorter and more similar to the other lines we have above :+1:

---

Oops, I implemented only half of your proposal. I'll fix.